README.md

```diff
@@ -10,15 +10,15 @@ Percona XtraBackup 8.0 can back up data from InnoDB, XtraDB, MyISAM, and MyRocks
 
 Percona XtraBackup 2.4 can back up data from InnoDB, XtraDB, and MyISAM tables on MySQL 5.7 servers, as well as Percona Server for MySQL with XtraDB.
 
-For more information about the Percona XtraBackup components and how to use this product, see the [Percona XtraBackup documentation](https://www.percona.com/doc/percona-xtrabackup).
+For more information about the Percona XtraBackup components and how to use this product, see the [Percona XtraBackup documentation](https://docs.percona.com/percona-xtrabackup/).
```

Comment:
The older link works. Is it being replaced?

Reply:
The old links work because of redirects added by IT team. Since other docs and blog posts can contain the old link https://www.percona.com/doc/percona-xtrabackup

Reply:
Good enough. Thanks.